api/controllers/OpenPaperAppController.js

```diff
@@ -1,6 +1,6 @@
 const SummaryController = require('./SummaryController');
 
-const OpenAppController = {
+const OpenPaperAppController = {
```

Comment:
Fair point, the reason this was names is because the paper and eApps have different views from the dashboard.

This is what an eApp looks like:
<img width="984" alt="Screenshot 2022-03-29 at 17 37 20" src="https://user-images.githubusercontent.com/1377253/160662600-092a2b56-9e7f-4fe9-a348-e518c8ae0b8e.png">


And this is what a paper app looks like:

<img width="919" alt="Screenshot 2022-03-29 at 17 40 30" src="https://user-images.githubusercontent.com/1377253/160662650-688f4f3e-f8a4-4858-989e-0ea28677cb47.png">

We have plans to change them to make them more similar but now they look quite different.